config/routes.rb

```diff
@@ -161,8 +161,8 @@
         # People
         # get "admin/users", to: ""
```

Comment:
Hmm I think something is missing here, but I don't have the full context haha 😅 @Ridhwana should I leave this for you?

The bug encountered is:
1. enable the `admin_restructure` feature flag
2. go to `localhost:3000/admin/users/1`
3. see that it fails

Reply:
Thanks for the heads up @Zhao-Andy. I dropped a note in slack :) - https://forem-team.slack.com/archives/C7GE84DEJ/p1616051436034400?thread_ts=1616022371.027000&channel=C7GE84DEJ&message_ts=1616051436.034400